tests/test_contract_special_part_release.py

```
@@ -0,0 +1,69 @@
+        procedural_requirements=[],
+        verified_claims=lines,
+        unverified_claims=[],
+        source_urls=["https://adilet.zan.kz/rus/docs/K990000409_"],
+        notes=[],
+    )
+
+
+def test_services_contract_injects_only_verified_special_part_articles():
+    research = _research([
+        "По договору возмездного оказания услуг исполнитель оказывает услуги, а заказчик их оплачивает "
+        "[основание: статья 683 ГК РК (Особенная часть); текст нормы: исполнитель обязуется оказать услуги, а заказчик обязуется оплатить эти услуги; источник: https://adilet.zan.kz/rus/docs/K990000409_]",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)